core/ledger/ledgerstorage/store.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/hyperledger/fabric/core/ledger/pvtdatastorage"
 )
 
-const maxBlockFileSize = 64 * 1 * 1024
+const maxBlockFileSize = 16 * 1 * 1024
```

Comment:
Why are we reducing the max block file size from 64kb to 16kb?

Reply:
It's just for testing purpose. We'll revert it later on.

Reply:
ok